docs/INSTALL.md

```diff
@@ -227,6 +227,17 @@ export PATH=$HOME/Android/Sdk/cmake/3.18.1/bin:$PATH
 export PATH=$HOME/Library/Android/Sdk/cmake/3.18.1/bin:$PATH
 ```
 
+Check if you have a system-wide Java runtime environment (JRE) installed:
+```bash
+java --version
+```
+
+If your system doesn't have a JRE installed or you want command line builds to use a JRE version bundled with the Studio then set `JAVA_HOME` environment variable:
```

Comment:
For Mac OS:
```bash
java -version
```
 ... or Java version is less than 11 (openjdk) ..
```bash
JAVA_HOME=*A-PATH-TO-YOUR-ANDROID-STUDIO-INSTALLATION*/Contents/jre/Contents/Home
```

Reply:
Thanks, I'll fix that!
Actually in linux its also possible to use `java -version` (single dash)